docs/pages/guides/keyboard-handling.mdx

```diff
@@ -0,0 +1,203 @@
+  const height = useSharedValue(0);
+
+  useKeyboardHandler(
+    {
+      onMove: e => {
+        'worklet';
+        height.value = Math.max(e.height, 0);
+      },
+    },
+    []
+  );
+  return { height };
+};
+```
+
+The above example uses the `useSharedValue` hook from Reanimated to store the keyboard's height. The value changes from 0 when the keyboard is dismissed to the maximum height when it's active.
+
+Now, let's use our `useGradualAnimation` hook inside a component:
+
+```js
+export default function TabTwoScreen() {
+  const { height } = useGradualAnimation();
+
+  const fakeView = useAnimatedStyle(() => {
+    return {
+      height: Math.abs(height.value),
+    };
+  }, []);
+
+  return (
+    <View style={styles.container}>
+      <FlatList
+        data={messages}
+        renderItem={({ item }) => <MessageItem message={item} />}
+        keyExtractor={item => item.createdAt.toString()}
+        contentContainerStyle={styles.listStyle}
+      />
+      <TextInput placeholder="Type a message..." style={styles.textInput} />
+      <Animated.View style={fakeView} />
+    </View>
+  );
+}
+```
+
+This example uses the keyboard's height to move the screen content above the keyboard. The view after the keyboard has an animated style that modifies its height based on the keyboard's height.
+
+### Handling multiple inputs
+
+For screens with multiple inputs, Keyboard Controller provides `KeyboardAwareScrollView` and `KeyboardToolbar` components. These handle input navigation and prevent the keyboard from covering the screen without custom configuration:
+
+```js
+import { KeyboardAwareScrollView, KeyboardToolbar } from 'react-native-keyboard-controller';
+
+export default function AdvancedToolbar() {
+  return (
+    <>
+      <KeyboardAwareScrollView bottomOffset={62} contentContainerStyle={styles.container}>
+        <View>
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+        </View>
+        <TextInput placeholder="Type a message..." style={styles.textInput} />
+        <View>
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+        </View>
+        <TextInput placeholder="Type a message..." style={styles.textInput} />
+      </KeyboardAwareScrollView>
+      <KeyboardToolbar />
+    </>
+  );
+}
+```
+
+The example wraps the inputs with `KeyboardAwareScrollView` to prevent the keyboard from covering them. The `KeyboardToolbar` component displays navigation controls and a dismiss button. While it works without configuration, you can customize the toolbar content if needed.
+
+Keyboard Controller provides consistency across Android and iOS with minimal configuration, offering the native feel users expect.
```

Comment:
```suggestion
Keyboard Controller provides consistency across Android and iOS with minimal configuration, offering the native feel that users expect.
```

Let's move this line to the introduction of the Keyboard Controller section since it is telling about the advantage that this library provides.